### PR TITLE
README update!

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
-This repository is meant for all community events organized by 🤗 HuggingFace.
+A central repository for all community events organized by 🤗 HuggingFace. Come one, come all!
+We're constantly finding ways to democratise the use of ML across modalities and languages. This repo contains information about all past, present and upcoming events.
 
-## Content
+## Hugging Events
 
-May 16 - May 31: [Blocks Party 🎊](gradio-blocks)
+| **Event Name**                                                          | **Dates**       | **Status**                                                                                                   |
```
Comment (osanseviero) [inline README.md:8]: Maybe add the year in the dates column noo

**Review by osanseviero — APPROVED** [review]
Very cool, thanks for taking care of this! 🔥

File: README.md (hunk 1 above)
Comment (Vaibhavs10) [reply README.md:8]: Good idea.. doing that right now.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+|-------------------------------------------------------------------------|-----------------|--------------------------------------------------------------------------------------------------------------|
+| [Whisper Fine Tuning Sprint](/whisper-fine-tuning-sprint)               | Dec 5th - 19th, 2022  | [Registrations open](https://discuss.huggingface.co/t/open-to-the-community-whisper-fine-tuning-event/26681) |
```
Comment (sanchit-gandhi) [inline README.md:10]: (renamed this to "Whisper Fine Tuning Event" and "/whisper-fine-tuning-event")

Comment (sanchit-gandhi) [reply README.md:10]: See:
https://github.com/huggingface/community-events/pull/39/commits/e3f0554373bd272543cb21ce1d789fb93c650212

Comment (Vaibhavs10) [reply README.md]: Fixed! Thanks for catching that! 🤗

**Pull request merged**